src/bodypart.cpp

```diff
@@ -65,7 +65,7 @@ std::string body_part_name (body_part bp)
     case bp_foot_r:
         return _("right foot");
     default:
-        return _("appendix");
+        return _("appendage");
```

Comment:
It's probably meant as a joke, but I suppose it's much better to return an empty string and print `debugmsg()` to indicate a possible hidden bug more explicitly. Otherwise we're likely to receive a "funny" issue report some day. Also, the "secret joke" needlessly increases amount of work for the translators. Same below.